1.0 release (#606)
zeus-health/ctw-component-library: .changeset/lovely-oranges-hammer.md

---
"@zus-health/ctw-component-library": major
---

> Given that we are now in production with several customers and the maturity of where we feel we are at now we are releasing 1.0!

Author: ```suggestion
Given where we are with maturity and stability we are now ready to release 1.0!
```
